04_numeronym/sahidakhtar1/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, len(vals))
+	for i, v := range vals {
+		result[i] = createNumeronym(v)
+	}
+	return result
+}
+
+func createNumeronym(s string) string {
+	trimedString := strings.TrimSpace(s)
+	var numeronym string
+	//If the string contains space or \t or \n then it will return the same string
+	if strings.ContainsAny(trimedString, " & \t &\n") {
+		return s
+	}
+	runes := []rune(trimedString)
+	count := len(runes)
+	if count <= 3 {
+		numeronym = trimedString
+	} else {
+		numeronym = string(runes[0]) + strconv.Itoa(count-2) + string(runes[len(runes)-1])
```

Comment:
The last term here: `string(runes[len(runes)-1])`, you have already stored `len(runes)` as `count` so you can write `string(runes[count-1])`.
Not a big issue but makes it a little more readable.

Reply:
Done